File: lib/simple_pvr/pvr_logger.rb
require 'logger'

module SimplePvr
  class PvrLogger
    @@logger = Logger.new(STDOUT)
    @@logger.formatter = proc do |severity, datetime, progname, msg|
      "[#{datetime.strftime('%Y-%m-%d %H:%M:%S')}] #{severity} #{msg}\n"
    end
  
    def self.info(message)
      @@logger.info(message)
    end
  end
end

File: python/simple_pvr/pvr_logger.py
import logging

def logger():
    return PvrLogger()

class PvrLogger:
    logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)

    def __init__(self):
        self.logger = logging.getLogger('PvrLogger')

    def info(self, message):
        self.logger.info(message)

    def warn(self, message):
        self.logger.warning(message)

    def debug(self, message):
        self.logger.debug(message)

    def error(self, message):
        self.logger.error(message)